cfgov/unprocessed/apps/retirement/js/views/graph-view.js

```diff
@@ -107,7 +106,7 @@ function init() {
     moveIndicatorToAge($(this).attr('data-age-value'));
   });
 
-  $('[data-bar_age]').click(function () {
+  $('#claim-canvas').on('click', '[data-bar_age]', function () {
```

Comment:
What even is `[data-bar_age]` doing in this call? In the original jquery `on` method docs https://api.jquery.com/on/, it shows the second argument as being passed to the `event.data` object, but this object isn't referenced in the event handler function here, and in fact, the attribute name is looked up independent of the value of this argument. 

Could this whole thing be re-written to something like this?

```js
const claimCanvasDom = document.querySelector('#claim-canvas');
claimCanvasDom.addEventListener('click', function() {
   const age = claimCanvasDom.getAttribute('data-bar_age');
   moveIndicatorToAge(age);
});
```